components/fxa-client/ios/FxAClient/FirefoxAccount.swift

```diff
@@ -332,6 +411,19 @@ public struct AccessTokenInfo {
         key = msg.hasKey ? ScopedKey(msg: msg.key) : nil
         expiresAt = Date(timeIntervalSince1970: Double(msg.expiresAt))
     }
+
+    // For testing.
+    internal init(
+        scope: String,
+        token: String = "toktok",
```

Comment:
I have a very mild objection to providing mock test data in a default arg like this, for something that is not itself in a test file. Could we default it to e.g. the empty string if it must have a default?